mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -573,9 +571,6 @@ class _DashboardViewState extends State<DashboardView>
 
     context.read<FavouritePlaceBloc>().add(const SyncFavouritePlaces());
     context.read<LocationHistoryBloc>().add(const SyncLocationHistory());
-    context.read<KyaBloc>().add(const FetchQuizzes());
```

Comment:
Hi @BenjaminSsempala , please tell me more about this removed code. Just share as if you are onboarding me. Thanks!

Reply:
Hello @Baalmart 
I'd added this in the translation PR such that the lessons and Quiz are translated immediately when language is changed, in that when the language is changed, the quiz endpoint is called again with the new parameter for language, something like `{AirQoUrls.kya}/quizzes/users/$userId?language=fr`
This introduced a bug of sorts, in that the quiz/kya was being fetched continuously before the progress on the app was synced, so the progress and progress bar were behaving unexpectedly. 

The solution I came up with was to clear the KYA and Quiz and then recall them, which would only be done at startup or restart of the app, and the change for that is also in this PR
` context.read<KyaBloc>().add(const ClearKya());
    context.read<KyaBloc>().add(const ClearQuizzes());`

Reply:
thanks @BenjaminSsempala